src/main/java/io/percy/selenium/Percy.java

```diff
@@ -87,8 +88,8 @@ public void snapshot(String name) {
      * @param widths The browser widths at which you want to take the snapshot. In
      *               pixels.
      */
-    public void snapshot(String name, List<Integer> widths) {
-        snapshot(name, widths, null, false, null, null);
+    public JSONObject snapshot(String name, List<Integer> widths) {
+        return snapshot(name, widths, null, false, null, null, false);
```

Comment:
no need to add return as well when sync is not passed + don't pass default value as false for sync since it will override global config

Reply:
resolved